ng/wasm/src/frost.rs

```diff
@@ -0,0 +1,420 @@
+use std::{convert::TryInto as _, iter::zip};
+
+use frost_core::{
+    aggregate,
+    keys::{PublicKeyPackage, VerifyingShare},
+    round1::{NonceCommitment, SigningCommitments},
+    round2::SignatureShare,
+    Ciphersuite, Identifier, Signature, SigningPackage, VerifyingKey,
+};
+use js_sys::Uint8Array;
+use serde::Deserialize;
+use serde_bytes::Bytes;
+use tsify::Tsify;
+use wasm_bindgen::prelude::*;
+
+use crate::abi::{from_js, into_js, JsResult};
+
+#[derive(Tsify, Deserialize)]
+#[tsify(from_wasm_abi)]
+pub enum FrostVariant {
```

Comment:
Again why are we copying code from lit-frost? The point of a crate is that its reusable.